.gitattributes

```diff
@@ -1,4 +1,4 @@
 # GitHub syntax highlighting
 pixi.lock linguist-language=YAML
 # Github doesn't recognize the docs folder as a documentation folder
-docs/* linguist-documentation=false
+docs/**/* linguist-documentation=false
```

Comment:
At this stage, we can remove the comments and sort the entries

Reply:
Aye